recipes/m2-recipes/meta.yaml

```diff
@@ -0,0 +1,108 @@
+package:
+  name: m2-binary-packages
+  version: 20230914
```

Comment:
Since this epoch appears all over, it might be a good idea to set a variable and reuse that?

Reply:
This file is auto-generated by the script given. I don't intend to make changes to it.